openshift/release/release-notes.md

```diff
@@ -0,0 +1,376 @@
+# OpenShift Pipelines TP-1.3
+
+# Tekton Pipelines v0.19.0
+
+## Features
+
+* :sparkles: Do not allow use of deprecated Conditions with custom tasks ([#3601](https://github.com/tektoncd/pipeline/pull/3601))
+
+  Disallow use of Conditions in pipeline tasks that reference custom tasks
+
+* :sparkles: Plumb the TEKTON_RESOURCE_NAME env var into the cluster and PR reso… ([#3555](https://github.com/tektoncd/pipeline/pull/3555))
+
+  Binary file (standard input) matches
+
+* :sparkles: Plumb the TEKTON_RESOURCE_NAME environment variable into more resou… ([#3532](https://github.com/tektoncd/pipeline/pull/3532))
+
+  Binary file (standard input) matches
+
+* :sparkles: Add variable expansion for ImagePullPolicy in Tasks. See #3423 ([#3488](https://github.com/tektoncd/pipeline/pull/3488))
+
+  Add variable expansion in Tasks for fields:
+
+  * spec.steps[].imagePullPolicy
+  * spec.sidecar[].imagePullPolicy
+* :sparkles: Integrate custom tasks into Pipelines ([#3463](https://github.com/tektoncd/pipeline/pull/3463))
```

Comment:
we can remove this custome atsk as discussed over [slack](https://coreos.slack.com/archives/CG5GV6CJD/p1611157583221600?thread_ts=1611156521.216800&cid=CG5GV6CJD)